src/Microsoft.ServiceFabric.Actors.Tests/Generator/ActorNameFormatTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.ServiceFabric.Actors.Generator;
```

Comment:
General comment applies to all .cs files in this PR: Please add the copyright header and move using namespace within namespace below.